tests/integration/test_requirements_dev.py

```
@@ -257,14 +273,209 @@
+        )
+        assert 'types-PyYAML' in package_names, (
+            "types-PyYAML should be present in requirements-dev.txt (added in this branch)"
+        )
+        
+        # Both should have version constraints
+        pyyaml_entry = next((ver for pkg, ver in requirements if pkg == 'PyYAML'), None)
+        types_entry = next((ver for pkg, ver in requirements if pkg == 'types-PyYAML'), None)
+        
+        assert pyyaml_entry and '>=6.0' in pyyaml_entry, (
+            "PyYAML should have version constraint >=6.0"
+        )
```

Ruff (0.14.6)

Assertion should be broken down into multiple parts

(PT018)